src/renderer/components/monaco-editor/get-editor-height-from-lines-number.injectable.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import { getInjectable } from "@ogre-tools/injectable";
+
+export type MonacoEditorSize = (linesCount: number) => number;
+
+const getEditorHeightFromLinesCountInjectable = getInjectable({
```

Comment:
So why in general do we need this **function/feature?** to be `*.injectable.ts`? 
Any useful examples of how it would be utilized later in other envs? Or what's purpose in other words?

Reply:
It is implemented as injectable to follow de-facto standard and being able to inject it as dependency into `MonacoEditor`:

```
export const MonacoEditor = withInjectables<Dependencies, MonacoEditorProps, MonacoEditorRef>(
  React.forwardRef<MonacoEditorRef, MonacoEditorProps & Dependencies>((props, ref) => <NonInjectedMonacoEditor innerRef={ref} {...props} />),
  {
    getProps: (di, props) => ({
      ...props,
      userStore: di.inject(userStoreInjectable),
      activeTheme: di.inject(activeThemeInjectable),
      getEditorHeightFromLinesCount: di.inject(getEditorHeightFromLinesCountInjectable),
    }),
  },
);
```

Without `injectable` format it would be used from the shared state (eg. imported module) which is considered bad practice.

Reply:
Alright, but why you've decided to control only height from MonacoEditor and this is why it's moved to `injectable.ts`?
You still didn't answer my question: if you able to control height of the editor, when/where would reuse it? 
Some examples? Maybe builds for `lens-ide` or `lens-some-build` with different initial heights for the editor? This would make sense?